docs/dev-notes/2026-01-02/migrate-from-svelte-5-ui-lib-to-flowbite-svelte/testing-strategy.md

```
@@ -0,0 +1,490 @@
+| **フェーズ2**           | 全テスト実行による回帰検出        | `pnpm test:integration`       | 毎回の CI 実行          |
+
+---
+
+## 前提条件確認（フェーズ-1）
+
+フェーズ実装前に、以下を手動確認してセレクタを特定します。
+
+### 前提条件確認の流れ
+
+1. `pnpm dev` で開発サーバ起動
+2. ブラウザで http://localhost:5174/ にアクセス
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)